src/main/java/com/imsweb/datagenerator/RandomTextGeneratorUtil.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2023 Information Management Services, Inc.
+ */
+package com.imsweb.datagenerator;
+
+import java.util.Random;
+
+public final class RandomTextGeneratorUtil {
+
+    private static final String[] _FILLER_TEXT = new String[] {
+            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+            "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+            "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
+            "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    };
+    private static final Random _RANDOM = new Random();
```

Comment:
I don't remember the exact reason, but RandomUtils uses a "SecuredRandom" object instead of a Random one. I am pretty sure it was done to fix some warnings about vulnerability, and I think we should do that here as well.

On a side note, this is a tiny class with a single method; I would have added it to RandomUtils. But since you already created the class, we can leave it alone. It's possible it will grow over time, who knows.